deeprank/features/atomicfeature_1.py

```diff
@@ -0,0 +1,703 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
```

Comment:
I suppose this piece of commentary should be changed to something informative about the feature being provided.

Reply:
Yes, i forgot to change this one.